src/components/MainContent/index.js

```diff
@@ -53,13 +55,18 @@ function MainContent() {
                 Submit
             </button>
             <div className={styles.singleCoinDiv}>
-                {/* Use redux favorites slice to indicate if there was no data returned after fetch */}
+                {/* Use redux favorites slice to indicate if there was no data
```

Comment:
Same. This sort of comment isn't needed. Psuedo code that you should keep is stuff that isn't self documenting code, that is steps to what the function is/should do, or a necessary explanation of why a variable is there